app/lib/actions/pulp3/orchestration/content_view_version/export.rb

```diff
@@ -18,7 +18,9 @@ class Export < Actions::EntryAction
           end
 
           # rubocop:disable Metrics/MethodLength
-          def plan(content_view_version, destination_server:, chunk_size: nil, from_history: nil)
+          def plan(content_view_version:, destination_server: nil,
+                   chunk_size: nil, from_history: nil, validate_incremental: true)
+            content_view_version = ::Katello::ContentViewVersion.find(content_view_version) if content_view_version.is_a? Integer
```

Comment:
Checking for data type in this case seems strange. Can you just check content_view_version.present?